requirements.txt

```diff
@@ -0,0 +1,55 @@
+antlr4-python3-runtime==4.7.2
+asttokens==2.4.1
+Cartopy==0.23.0
+certifi==2024.6.2
+cf-units==3.2.0
+cftime==1.6.4
+click==8.1.7
+cloudpickle==3.0.0
+contourpy==1.2.1
+coverage==7.6.0
+cycler==0.12.1
+dask==2024.6.0
+decorator==5.1.1
+executing==2.0.1
+f90nml==1.4.4
+fonttools==4.53.0
+fsspec==2024.6.0
+iniconfig==2.0.0
+ipython==8.25.0
+jedi==0.19.1
+Jinja2==3.1.4
+kiwisolver==1.4.5
+locket==1.0.0
+MarkupSafe==2.1.5
+matplotlib==3.9.0
+matplotlib-inline==0.1.7
+mule @ git+https://github.com/metomi/mule@cce4b99c7046217b1ec1192118a786636e0d8e54#subdirectory=mule
+netCDF4==1.7.1
+numpy==1.26.4
+packaging==24.1
+parso==0.8.4
+partd==1.4.2
+pexpect==4.9.0
+pillow==10.3.0
+pluggy==1.5.0
+prompt_toolkit==3.0.47
+ptyprocess==0.7.0
+pure-eval==0.2.2
```

Comment:
`conda` thinks this is called `pure_eval`... weird. (The requestion version is available though.)